### PR TITLE
Update FAQ block type code example

### PR DESCRIPTION
**Pull request by mliebischer**
The example code for the faq block type was outdated. Those are the same fixes as in https://github.com/rohberg/volto-accordion-block/issues/1

### CODE REVIEW

File: docs/mastering-plone/volto_custom_addon2.md
Comment (stevepiercy) [inline docs/mastering-plone/volto_custom_addon2.md:460]: We want to keep matching/closing backticks together to avoid whitespace.

```suggestion
```

**Review by stevepiercy — APPROVED** [review]
LGTM with a minor change.

Someone familiar with this training's technical details should review before merging.

File: docs/mastering-plone/volto_custom_addon2.md
Comment (stevepiercy) [inline docs/mastering-plone/volto_custom_addon2.md:296]: Good catch!

Now if we could only get that code sample to have syntax highlighting. We have noticed that if the syntax is truly not `jsx`, then pygments cannot highlight it properly. Do you see any syntax issue in the example?

**Review by stevepiercy — APPROVED** [review]

File: docs/mastering-plone/volto_custom_addon2.md
Comment (ksuess) [reply docs/mastering-plone/volto_custom_addon2.md:296]: The lines are hightlighted. Maybe a local problem?

File: docs/mastering-plone/volto_custom_addon2.md
Comment (ksuess) [inline docs/mastering-plone/volto_custom_addon2.md:587]: I would like to keep the figure.

````
```{figure} _static/volto_addon_accordion_add.png
:alt: "@rohberg/volto-accordion-block"
```
````

**Review by ksuess — CHANGES_REQUESTED** [review]
Thanks for caring! 
With the figure brought back, I will approve.

**Comment by ksuess** [comment]
> I just noticed that the two test failures are from redirects in links.
> 
>     * https://github.com/plone/training/runs/7249210720?check_suite_focus=true#step:5:3035
> 
>     * https://github.com/plone/training/runs/7249210720?check_suite_focus=true#step:5:3082
> 
> 
> I could either fix them in a separate PR and you can merge `main` into your branch, or you can fix them as part of this PR. Please let me know. Thank you!

It was just a tempoararily broken link: https://github.com/plone/training/runs/7249210720?check_suite_focus=true#step:5:3086
With a re-run of the test it's fine now. 

Just ping us on Discord channel "documentation" https://discord.com/channels/786421998426521600/862642144401031188 if an unrelated test failure happens. 

A re-run of tests can be done by heading over to tab "actions" and restart failed tests like the linkcheck.
image

**Comment by stevepiercy** [comment]
Oh, that's right. We don't run `linkcheckbroken` with it erroring it out on either broken or redirect, like we do in `plone/documentation`. Maybe we should?

In any case, those two redirect links must be fixed, either in this PR or another.

File: docs/mastering-plone/volto_custom_addon2.md
Comment (stevepiercy) [reply docs/mastering-plone/volto_custom_addon2.md:296]: The line numbers needed to be shifted by 1. See the preview: https://deploy-preview-626--plone-training.netlify.app/mastering-plone/volto_custom_addon2.html

Screen Shot 2022-07-10 at 3 45 59 AM

Comment (mliebischer) [reply docs/mastering-plone/volto_custom_addon2.md:296]: Line emphasizing (the yellow one) should work, but JSX syntax highlighting is broken due to the `aria-label` attribute in Line 444:

``` jsx
<button aria-label={props.intl.formatMessage(messages.add)} onClick={addQA}>
   <Icon name={plusSVG} size="18px" />
</button>
``` 

It seems to be the same problem as described here https://github.com/plone/volto/issues/3038#issuecomment-1041390641. Using camelCase (`ariaLabel`) will result in compilation errors. I have no clue how to fix it..  🤔

File: docs/mastering-plone/volto_custom_addon2.md
Comment (mliebischer) [reply docs/mastering-plone/volto_custom_addon2.md:587]: Oops, the figure was accidentally deleted.

File: docs/mastering-plone/volto_custom_addon2.md
Comment (stevepiercy) [reply docs/mastering-plone/volto_custom_addon2.md:296]: Thank you for identifying the problem. It is in fact a [bug in jsx-lexer](https://github.com/fcurella/jsx-lexer/issues/15#issuecomment-1042808552), which is a plug-in for Pygments which lacks JSX support. Let's leave this code sample as is.

Comment (stevepiercy) [reply docs/mastering-plone/volto_custom_addon2.md:296]: I think this may be a simple fix by changing the regex for the [lexer's `Name.Attribute`](https://github.com/fcurella/jsx-lexer/blob/master/jsx/lexer.py#L30).

```python
([\w]+\s*)(=)(\s*)
```

to this:

```python
([\w-]+\s*)(=)(\s*)
```

Comment (stevepiercy) [reply docs/mastering-plone/volto_custom_addon2.md:296]: PR submitted: https://github.com/fcurella/jsx-lexer/pull/16

Comment (stevepiercy) [reply docs/mastering-plone/volto_custom_addon2.md:296]: I am resolving this bit, because I expect the PR in jsx-lexer to take some time to get merged.

**Comment by stevepiercy** [comment]
> In any case, those two redirect links must be fixed, either in this PR or another.

Done in #627.

@mliebischer once you restore the figure, we can merge. Thank you!

File: docs/mastering-plone/volto_custom_addon2.md
Comment (stevepiercy) [reply docs/mastering-plone/volto_custom_addon2.md:296]: Holy cow! Now with a new release!

https://github.com/fcurella/jsx-lexer/releases/tag/v2.0.0

**Pull request merged**
